### DISCUSSION TITLE
win7  zlibrary 安装失败

### DISCUSSION THREAD

**Discussion by butchercome** [tw93/Pake#292] [Q&A] [answered]
安装exe 软件时 失败 安装不成功 

there is a problem with this windows installer package a program run as part of the setup did not finish as expected concat your support personnel or package verder

**Comment by Tlntin** [comment, marked as answer]
win7未附带webview2，需要自行安装。https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section
